src/modules/file-preview-card.marko

```diff
@@ -0,0 +1,330 @@
+<div id="file-preview-card">
+    <section-header metadata=input.metadata name=input.name alpha="true"/>
+
+    <p>
+        The file-preview-card can be used to display file previews. A css grid layout is used to define the areas in which certain elements can be placed.
+    </p>
+
+    <h3>Uploading</h3>
+    <div class="demo">
+        <div class="demo__inner">
+            <div style="height: 168px; width: 168px;">
+                <div class="file-preview-card">
+                    <div class="file-preview-card__center">
```

Comment:
We  prefer semantic class names over presentational ones. So things like body, cell, main etc over "center".